meta-balena-common/classes/resin-u-boot.bbclass

```diff
@@ -48,3 +48,6 @@ python do_generate_resin_uboot_configuration () {
     bb.utils.copyfile(src, dst)
 }
 addtask do_generate_resin_uboot_configuration after do_patch before do_configure
+
+# Regenerate env_resin.h if any of these variables change.
+do_generate_resin_uboot_configuration[vardeps] = "RESIN_UBOOT_DEVICES RESIN_UBOOT_DEVICE_TYPES RESIN_BOOT_PART RESIN_DEFAULT_ROOT_PART RESIN_IMAGE_FLAG_FILE RESIN_FLASHER_FLAG_FILE RESIN_ENV_FILE BASE_OS_CMDLINE"
```

Comment:
Could we avoid this duplication of variable names in the python function and in this? Maybe defining a list of variable names that is consumed by both?

Reply:
Changed to use a single list consumed by both.